code/game/objects/effects/misc.dm

```diff
@@ -137,3 +137,19 @@
 	. = ..()
 	icon_state = "snap3"
 	render_target = "*snap[id]"
+
+/obj/effect/frosty_breath
+	icon = 'icons/effects/effects.dmi'
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+	layer = ABOVE_MOB_LAYER
+	anchored = TRUE
+
+/obj/effect/frosty_breath/Initialize(mapload, mob/living/carbon/C)
+	. = ..()
+	dir = C.dir
+	if(C.buckled) //dont need to offset when they are upsidedown+buckled (on meat spike)
+		pixel_y = (C.buckled.buckle_offset + 16)
+	if(dir == NORTH)
+		layer = BELOW_MOB_LAYER
+	flick("breath_[istype(C.buckled, /obj/structure/kitchenspike) ? "180" : "[C.lying_prev]" ]", src) //you can thank the kitchenspike code not using lying_angle for this jank
+	QDEL_IN(src, 2 SECONDS)
```

Comment:
why use different sprites instead of just rotating the same sprite using a matrix transformation?

Reply:
L

Reply:
`flick` doesn't support matrix transforms. It's only `icon_states` or raw `dmi`s.

Reply:
![image](https://user-images.githubusercontent.com/16618648/178282987-ae2d130d-d862-4df3-bc9f-d83d770a3391.png)

maybe this will illustrate everything better